rllib/policy/torch_policy.py

```diff
@@ -212,18 +214,17 @@ def optimizer(self):
 
     def _lazy_tensor_dict(self, postprocessed_batch):
         train_batch = UsageTrackingDict(postprocessed_batch)
-
-        def convert(arr):
-            if torch.is_tensor(arr):
-                return arr.to(self.device)
-            tensor = torch.from_numpy(np.asarray(arr))
-            if tensor.dtype == torch.double:
-                tensor = tensor.float()
-            return tensor.to(self.device)
-
-        train_batch.set_get_interceptor(convert)
+        train_batch.set_get_interceptor(self._convert_to_tensor)
         return train_batch
 
+    def _convert_to_tensor(self, arr):
```

Comment:
Should we move these to `utils` and make them non-private? Could be useful anywhere and avoids possible circular import problems.